src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/ServiceDescriptionService.java

```diff
@@ -225,7 +225,7 @@ public ServiceDescriptionType addWsdlServiceDescription(ClientId clientId, Strin
             UnhandledWarningsException,
             ServiceAlreadyExistsException,
             InvalidUrlException,
-            WsdlUrlAlreadyExistsException {
+            WsdlUrlAlreadyExistsException, InterruptedException {
```

Comment:
Would it be good to add javadoc @throws documentation explaining where these InterruptedExceptions can originate from? At least to the public main-level service methods (addWsdlServiceDescription, updateWsdlUrl)?

Reply:
Fixed a few javadocs